test/unit_test.py

```diff
@@ -1,6 +1,8 @@
+from decimal import Decimal
+
 import pytest
```

Comment:
shouldn't `import pytest` go on the first line?

Reply:
Yup, see my comment. In this case, since they separated into 2 groups:
1. system, python-level default libs -> 'decimal'
2. external python package -> 'pytest'

This allows for easier distinguishing of sys vs. external imports